### DISCUSSION TITLE
Found copied code

### DISCUSSION THREAD

**Discussion by PrajaktaSathe** [community/community#23197] [New to GitHub]
<p>Hi there!<br>
I don’t know if this is the right section to post my question. But here it goes.<br>
I have a repository in which I had opened an issue asking for some code on a certain topic. A person opened up a PR regarding that issue. But the code that was contributed was the exact same as that of a code that found on another website on Google. What should I do? :thinking:<br>
Thanks!</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>If you’re wondering if you’re allowed to merge the code, that generally depends on whether the code is copyrighted, and if so if it’s licensed in a way that allows you to use it (and under which conditions).</p>
<p>I’m afraid anything more specific would quick get too close to legal advice, for which you’d need to ask a lawyer.</p>
